include/view.h

```diff
@@ -129,18 +146,13 @@ void view_move_resize(struct view *view, struct wlr_box geo);
 void view_move(struct view *view, int x, int y);
 void view_moved(struct view *view);
 void view_minimize(struct view *view, bool minimized);
-struct output *view_output(struct view *view);
 void view_store_natural_geometry(struct view *view);
-/* output is optional, defaults to current nearest output */
-void view_center(struct view *view, struct output *output,
-	const struct wlr_box *ref);
+void view_center(struct view *view, const struct wlr_box *ref);
```

Comment:
When reading the function definition it doesn't immediately become clear to me what `*ref` means.
So maybe we could add either a comment above explaining it or rename the argument. Maybe something like `container` or `parent` would make it more clear?

Reply:
I'll add a comment, seems worth explaining.